challenges/functions.js

```diff
@@ -23,7 +23,7 @@
 
 // ==== Closures ==== 
 
-// Explain in your own words why nestedfunction can access the variable internal.
+// Explain in your own words why the nested function can access the variable internal.
```

Comment:
What do you think about changing to "nestedFunction" (including the quotes) instead, to make it completely clear what variable we are referring to?

Reply:
oh! it was the function name! :open_mouth: I will fix it later.

Reply:
Fixed with f0d88af0e756aa306cca4a1c9c8412f0923abba0

Reply:
Nice.